client/src/elm/Pages/Utils.elm

```diff
@@ -1185,3 +1185,28 @@ customSaveButton language active msg label =
         , onClick msg
         ]
         [ text <| translate language label ]
+
+
+customPopup : Language -> Bool -> TranslationId -> ( Html msg, Html msg, msg ) -> Html msg
+customPopup language showWarning actionLabel ( topMessage, bottomMessage, action ) =
+    div [ class "ui active modal diagnosis-popup" ]
+        [ div [ class "content" ] <|
+            [ div [ class "popup-heading-wrapper" ]
+                [ img [ src "assets/images/exclamation-red.png" ] []
+                , div [ class "popup-heading" ] [ text <| translate language Translate.Warning ++ "!" ]
```

Comment:
Shouldn't the exclamation mark be part of the `translate.Warning`?

Reply:
Well,  we can make it whatever we like, but as I see it, we translate a word.
So, no punctuation marks need to be part of it.